### PR TITLE
Bug 8 Volunteers badges

### PR DESCRIPTION
**Pull request by filipgorczynski**
https://github.com/stxnext-csr/volontulo/issues/8

### CODE REVIEW

File: volontulo/templates/organizations/organization_view.html [hunk 1]
```diff
@@ -6,19 +6,19 @@
...
             <div class="col-xs-4 form-control-static">
                 <span>{{ organization.address }}</span>
             </div>
         </div>
         <div class="form-group form-group-sm">
-            <label class="col-xs-2 control-label" for="{{ organization_form.description.id_for_label }}">Opis</label>
-            <div class="col-xs-10 form-control-static">
-                <span>{{ organization.description }}</span>
+            <label class="col-xs-2 control-label">Opis</label>
+            <div class="col-xs-10">
```
Comment (Yog) [inline volontulo/templates/organizations/organization_view.html:20]: you have missed `form-control-static` class there. It is needed so label aligns well with the value

Comment (filipgorczynski) [inline volontulo/templates/organizations/organization_view.html:20]: Probably it aligns labels but os problematic with lot of text.

Comment (Yog) [inline volontulo/templates/organizations/organization_view.html:20]: this appears to be bootstrap bug

Comment (Yog) [inline volontulo/templates/organizations/organization_view.html:20]: this is a reason why we need to host our own styles
we could fix it then. 
Probably not in this task

**Comment by Yog** [comment]
Merging, bootsrap issues will be solved in #171

**Pull request merged**